pslab/external/BMP180.py

```diff
@@ -1,32 +1,24 @@
-from __future__ import print_function
```

Comment:
Woops, I accidentally deleted your comment @orangecms! To answer your question, this is indeed not needed. It was for python2 compatibility,

Reply:
Right. My comment actually refered to the numpy int16 thing. That looks like a huge dependency for such a little thing, WDYT?

Reply:
Ah, right. Yes, I can't see any reason why that would be necessary.

(I was trying to delete my own reply to your comment, but somehow managed to delete the whole thread. Sorry about that!)